prow/scripts/cluster-integration/kyma-aks-long-lasting.sh

```diff
@@ -207,7 +207,28 @@ function addGithubDexConnector() {
     export DEX_CALLBACK_URL="https://dex.${CLUSTER_NAME}.build.kyma-project.io/callback"
     go run "${KYMA_PROJECT_DIR}/test-infra/development/tools/cmd/enablegithubauth/main.go"
 }
+function getLetsEncryptCertificate {
```

Comment:
It should be moved to separate script. Please make it more generic.

Reply:
encrypt,decrypt, getlestencrypt and get-letsencrypt-tls were removed and put into helper scripts. aks/gke-long-lasting now call them

Reply:
The bash workload was requested